public/index.html

```diff
@@ -3,8 +3,116 @@
   <head>
     <meta charset="utf-8">
     <title>Powell Peralta Skateboards</title>
+    <link rel="stylesheet" href="/normalize.css">
+    <link rel="stylesheet" href="/app.css">
   </head>
   <body>
-
+    <main class="container">
+      <header>
+        <div class="top_connect">
+          <a class="sub_link" href="#">Subscribe</a>
+          <a class="social_link" href="https://www.facebook.com/powellperalta/?fref=ts">
+            <span class="fa fa-facebook"></span></a>
+          <a class="social_link" href="http://powell-peralta.com/rss/">
+            <span class="fa fa-rss"></span></a>
+          <a class="social_link" href="https://twitter.com/PowellPeralta">
+            <span class="fa fa-twitter"></span></a>
+          <a class="social_link" href="https://www.youtube.com/PowellPeralta">
+            <span class="fa fa-youtube"></span></a>
+        </div>
+        <div class="nav_items">
+          <ul>
+            <li>Sk8t News</li>
+            <li>Skateboarders</li>
+            <li>Board Design</li>
+            <li>Contact Us</li>
+          </ul>
+        </div>
+        <section class="hero">
+          <img class="hero_logo" src="img/peralta-logo.png" alt="Company Logo">
```

Comment:
I would probably group together these two images within the hero so that the logo can be positioned within the slider image and only one item has to be `position: absolute`